config/initializers/hesa_disabilities.rb

```diff
@@ -1,12 +1,26 @@
+module HesaDisabilityValues
```

Comment:
I'm in two minds about this. The use of constants has been used in order to have 'one source of truth'. This is useful in scenarios where a form value does not directly map onto a HESA value. On the other hand I'm worried this creates more indirection and makes the code less easier to understand. Any opinions on this welcome!

Reply:
If i understand correctly we now map twice — once from the form to the type, and then from the type to the code? How would it be if the form values were codes, then store just codes, no strings? Might need to override the labels in some cases.

Reply:
Yeah that's an interesting idea @duncanjbrown 

We could change the values of the radio buttons like follows, which would remove the need to have globals (via the initialisers) or any conversion:

```
          <%= f.govuk_radio_button :sex, {value: 'female', hesa_code: '00'}, label: { text: 'Female' }, link_errors: true %>
```
This would have implications for other parts of the code which are already storing the Hesa code via the conversion method (e.g. Subjects, Grades) i.e. refactoring - what do you think @tvararu ?

Reply:
Actually, that said, we would still need the conversion code when we come to backfilling ...

Reply:
The approach to save codes instead of strings makes sense, but let's do that only for these new HESA codes. The old ones can stay as they are and be revisited later as tech debt. :+1:

Reply:
Further work to add hesa codes to the forms has revealed a number of additional complexities, and results in far more changes as well as spreading hesa code logic in many places. @tvararu and I believe that it makes sense to stick with the original solution proposed in this PR, which is also consistent with previous implementations